src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1844,20 +1844,68 @@ When HTTPS is set up, you can automatically redirect any HTTP request from
 users to the secured version of the protocol to make sure exchanged information
 is protected. It requires to enable `web.redirect_http_to_https`.
 
```

Comment:
1) I prefer using a simpler sentence:
A default keystore is provided under `config/web/keystore` with the password `activeeon`.

2) This sentence should be removed because it is not correct:
It requires to enable `web.redirect_http_to_https`.

Reply:
ok